src/service/packet_router.proto

```diff
@@ -60,7 +74,30 @@ message packet_router_packet_down_v1 {
   window_v1 rx2 = 3;
 }
 
-service hotspot {
+
```

Comment:
I don't think the verifier report is needed and as mentioned in discord I'm not convinced yet that linking is quite the way to get easy auditing/reporting.

Reply:
Removed with commit 43d090e